allennlp/data/dataset_readers/squad.py

```diff
@@ -197,6 +200,8 @@ def text_to_instance(self,  # type: ignore
                 }
         if question_id:
```

Comment:
Looks like we can remove the question id now.

Reply:
I think it's nice to keep it there, e.g., I'm imagining using a version of this to dump individual scores per question into a file and then having easy access to the ID is handy.

Reply:
Ok, fair enough.